zitadel/local/README.md

```diff
@@ -36,9 +36,9 @@ admin users login:
 
 > Note: The first login requires a password change, please remember the new password.
 
-## Run Terraform Scripts to initialize the configuration for Applications.
+## Initialize the configuration for Applications.
```

Comment:
I see that many configurations, including the SMTP is valid not only for local but also for other enviornments. 
Infact, we need SMTP for non-local only. For local environments, we must show how to directly add users. 

Its better we copy the documentation or have one common documentation and have references from there to the local readme.md and to a new readme file directly under zitadel folder. 

Please note there is one more readme under terraform folder. 

If you wish, we can also merge all of them into one single readme folder directly under zitadel folder.

Reply:
Ok. I will merge all of them into the readme.md under the zitadel folder.